breathecode/marketing/serializers.py

```diff
@@ -137,13 +137,14 @@ def validate(self, data):
             raise ValidationException(f'Destination URL is invalid, returning status {status["status_code"]}')
 
         academy = Academy.objects.filter(id=self.context['academy']).first()
+
         if academy is None:
             raise ValidationException(f'Academy {self.context["academy"]} not found',
                                       slug='academy-not-found')
 
         utc_now = timezone.now()
-        days_ago = self.instance.created_at + timedelta(days=1)
-        if days_ago < utc_now:
+
+        if is_put and self.instance.created_at + timedelta(days=1) < utc_now:
```

Comment:
Test ShortLinkView https://github.com/jefer94/apiv2/blob/f28c0a14cc194faf559503f33b9c3da93a0cd997/breathecode/marketing/views.py#L440 the post and the put case